examples/hosts/app-integration/schema-upgrade/src/containerKillBit/containerKillBit.ts

```diff
@@ -44,36 +45,40 @@ export class ContainerKillBit extends DataObject implements IContainerKillBit {
         return this._taskManager;
     }
 
-    public get dead() {
-        return this.crc.read(deadKey) as boolean;
+    public get migrated() {
+        return this.crc.read(newContainerIdKey) !== undefined;
     }
 
-    public async setDead() {
+    public get newContainerId() {
+        return this._newContainerId;
+    }
+
+    public async setNewContainerId(id: string) {
         // Using a consensus-type data structure here, to make it easier to validate
-        // that the setDead was ack'd and we can have confidence other clients will agree.
-        return PerformanceEvent.timedExecAsync(
-            this.runtime.logger,
-            { eventName: "SettingContainerDead" },
-            async () => this.crc.write(deadKey, true).then(() => { }),
-        );
```

Comment:
These were used as a diagnosis tool when we were looking at the summarizer blocking the proposal acceptance.  Since we now have a handle on that issue (and it's fixed in this PR), we don't really need the diagnosis tool anymore.

Beyond that, the PerformanceEvent.timedExecAsync is an extra concept that the demo doesn't need, so it's nice to simplify potential distractions.  Here it would only be measuring CRC write performance anyway, which isn't super interesting to a demo author I think.